modules/perception/onboard/component/fusion_component.cc

```diff
@@ -185,8 +185,10 @@ bool FusionComponent::InternalProc(
 
   const double cur_time = apollo::common::time::Clock::NowInSeconds();
   const double latency = (cur_time - timestamp) * 1e3;
-  AINFO << "FRAME_STATISTICS:Obstacle:End:msg_time[" << timestamp
-        << "]:cur_time[" << cur_time << "]:cur_latency[" << latency
+  AINFO << "FRAME_STATISTICS:Obstacle:End:msg_time["
+        << std::to_string(timestamp)
```

Comment:
Does we really need to perform `std::to_string` first here?

Reply:
I add std::to_string to print complete digits of timestamp, otherwise it only shows scientific notation in logs, which is useless for debugging. Or is there another way to show more precision?

Reply:
@jeroldchen : you ,may consider std::setprecision() But please remember to unset appropriately.

Reply:
Have changed to std::setprecision().